src/ScenarioPreview/ScenarioPreview.tsx

```diff
@@ -0,0 +1,152 @@
+import { Box, Divider, Link, Stack, Typography } from "@mui/material";
+import { LabelChipGroup, LabelChipGroupProps } from "../LabelSelector";
+
+import DateLabel from "../DateLabel/DateLabel";
+import FileLabel from "../FileLabel/FileLabel";
+import FormatLabel from "../FormatLabel/FormatLabel";
+import FormatVersionLabel from "../FormatVersionLabel/FormatVersionLabel";
+import NoWrapTypography from "../NoWrapTypography/NoWrapTypography";
+import React from "react";
+import RoadLabel from "../RoadLabel/RoadLabel";
+import { ScenarioPreviewProps } from "./ScenarioPreview.types";
+import UserLabel from "../UserLabel/UserLabel";
+
+export function ScenarioPreview({
+  name,
+  href,
+  image,
+  description,
+  format,
+  formatVersion,
+  file,
+  createdAt,
+  user,
+  label = [],
+  roadLabel,
+  sx
+}: ScenarioPreviewProps) {
+  function hasOptionalProperty() {
+    return label?.length > 0 || createdAt || user;
+  }
+
+  const labelChips: LabelChipGroupProps["chips"] = label?.map(l => ({
+    clickable: false,
+    color: l.color,
+    label: l.name,
+    size: "small"
+  }));
+
+  return (
+    <Box
+      display="flex"
+      flexDirection="column"
+      gap={1}
+      minWidth={0}
+      fontFamily="Montserrat"
+      data-testid="scenario-preview-wrapper"
+      sx={{ ...sx }}
+    >
+      <Box>
+        <Stack direction="row" spacing={1} minWidth={0}>
+          <img
+            src={image}
+            alt="scenario-image"
+            style={{
+              height: "44px",
+              marginBottom: "auto",
+              marginTop: "auto",
+              objectFit: "contain",
+              width: "78px"
+            }}
+          />
+          <Stack direction="column" minWidth={0}>
+            <NoWrapTypography>
+              <Link
+                href={href}
+                target="_blank"
+                color="primary"
+                variant="subtitle2"
+                underline="hover"
+                data-testid="scenario-preview-name"
+              >
+                {name}
+              </Link>
+            </NoWrapTypography>
```

Comment:
This is what is causing the different color between the text and the truncation ellipsis. Suggestion should fix this. Also this removes the tooltip in case of overflow, which is the desired functionality.

```suggestion
              <Link
                href={href}
                target="_blank"
                color="primary"
                variant="subtitle2"
                underline="hover"
                textOverflow="ellipsis"
                overflow="hidden"
                noWrap
                data-testid="scenario-preview-name"
              >
                {name}
              </Link>
```